czech/global.lang.php

```diff
@@ -268,6 +268,11 @@
 $l['postbit_warning_level'] = "Úroveň varování:";
 $l['postbit_warn'] = "Varovat autora tohoto příspěvku";
 $l['postbit_purgespammer'] = "Odstranit spamera";
+$l['postbit_post_deleted'] = "Tento příspěvek byl smazán.";
+$l['postbit_post_unapproved'] = "Tento příspěvek čeká na schválení.";
+$l['postbit_thread_deleted'] = "Tohle vlákno bylo smazáno.";
+$l['postbit_thread_unapproved'] = "Tohle vlákno čeká na schválení.";
+$l['postbit_deleted_post_user'] = "Příspěvek od {1} byl smazán.";
```

Comment:
U toho posledního si nejsem jistý překladem, neboť 
"$l['postbit_deleted_post_user'] = "This post by {1} has been deleted.";
a byl bych si jistý až bych to viděl.